application/common/models/Method.php

```diff
@@ -324,4 +346,39 @@ public static function deleteExpiredUnverifiedMethods()
         }
     }
 
+    /**
+     * Gets all verified methods for user specified by $employeeId
+     * @param string $employeeId
+     * @return String[]
+     */
+    public static function getVerifiedMethods($employeeId)
+    {
+        $method = new Method;
+        return $method->idBrokerClient->listMethod($employeeId);
+    }
+
+    /**
+     * Gets a specific verified method for user specified by $employeeId
+     * @param string $uid
+     * @param string $employeeId
+     * @return null|String[]
+     * @throws \yii\web\NotFoundHttpException
+     * @throws \Exception
+     */
+    public static function getOneVerifiedMethod($uid, $employeeId)
+    {
+        $method = new Method;
+        try {
+            return $method->idBrokerClient->getMethod($uid, $employeeId);
```

Comment:
If broker returns unverified methods now this function either needs to be renamed or updated to check the verified status before returning it.

Reply:
@wcjr suggested leaving this alone since it isn't used in the UI, deferring the decision on whether to totally remove it.